src/c.tex

```diff
@@ -761,15 +767,21 @@ \subsection*{Integer Constant-Generation Instructions}
 \end{tabular}
 \end{center}
 C.LI loads the sign-extended 6-bit immediate, {\em imm}, into
-register {\em rd}.  C.LI is only valid when {\em rd}$\neq${\tt x0}.
+register {\em rd}.
 C.LI expands into {\tt addi rd, x0, imm[5:0]}.
+C.LI is only valid when {\em rd}$\neq${\tt x0};
+the code points with {\em rd}={\tt x0} encode HINTs.
```

Comment:
I guess I haven't actually read the ISA manual since hints were introduced, but I wouldn't say `C.LI is only valid when {\em rd}$\neq${\tt x0}` -- it's still valid, it's just a hint.  From reading the rest of this it looks like hints are being described as different instructions, so if that's the case then the wording is OK.